### PR TITLE
Update all browsers versions for File API

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects the real values for all browsers for the `File` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v4.0.0).

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/File

_Check out the [collector's guide on how to review this PR](https://github.com/foolip/mdn-bcd-collector#reviewing-bcd-changes)._

### CODE REVIEW

File: api/File.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://w3c.github.io/FileAPI/#file-section",
         "support": {
           "chrome": {
-            "version_added": "13"
+            "version_added": "1"
```
Comment (foolip) [inline api/File.json:9]: Shouldn't some of the members also be supported in Chrome 1?

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/File.json (hunk 1 above)
Comment (queengooborg) [reply api/File.json:9]: It looks like pretty much all the members are actually not defined in Chrome 13, even.  In these PRs, though, I'm only focusing on changes where the version number reports earlier than what BCD says, and leaving the other direction for another time -- I'll be sure to look into the members in the next wave!

Comment (foolip) [reply api/File.json:9]: If none of the members were present in Chrome 1, then there's a good chance that the `File` interface didn't actually work and should be `partial_implementation` and/or have some notes. I don't think we should update part of the data and leave it less consistent based just on a `"File" in self` test without understanding if the inconsistency is correct or not.

File: api/File.json [hunk 2]
```diff
@@ -148,7 +148,7 @@
               "version_added": "16"
             },
             "opera_android": {
-              "version_added": false
+              "version_added": "16"
```
Comment (foolip) [inline api/File.json:151]: Can you re-mirror all of the Opera data? It's inconsistent, Chrome/Android 13/18 versions don't map to the same Opera versions on different features.

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/File.json [hunk 3]
```diff
@@ -197,7 +197,7 @@
               "version_added": "10"
             },
             "opera": {
-              "version_added": "16"
+              "version_added": "≤15"
             },
             "opera_android": {
               "version_added": false
```
Comment (foolip) [inline api/File.json:203]:
```suggestion
              "version_added": "≤14"
```

I guess?

File: api/File.json [hunk 4]
```diff
@@ -303,7 +303,7 @@
...
```
Comment (foolip) [inline api/File.json:309]:
```suggestion
              "version_added": "≤14"
```

**Review by foolip — CHANGES_REQUESTED** [review]

**Review by foolip — APPROVED** [review]

**Pull request merged**